### ISSUE TITLE
Build error with gatsby

### ISSUE THREAD

**Issue by foxbit19** [oliverox/React-TypeMe#3]
Hi,
first of all: this package is awesome, thanks for your work.

I've encountered a problem when I used it on Gatsby. 
When I launch the command `gatsby build` it gives me the following error:
```
Generating SSR bundle failed

Unexpected token (109:14)

File: node_modules\react-typeme\src\TypeMe.js:109:14
```

[This line](https://github.com/oliverox/React-TypeMe/blob/master/src/TypeMe.js#L109) causes the problem but I can't understand why.

**Comment by oliverox** [comment]
Thanks for the comment. I'll have a look at it.

**Comment by oliverox** [comment]
Just did a quick test inside a new Gatsby project. TypeMe rendered without any errors. Any chance you can reproduce the error on a public site so I can have a deeper look?

**Comment by oliverox** [comment]
Oh wait. I see the issue. It occurs when building...

**Comment by foxbit19** [comment]
> Oh wait. I see the issue. It occurs when building...

Yes that's true. It occurs only when building. It works perfectly in `develop` "mode".

**Comment by oliverox** [comment]
Found the issue. Will push a fix tomorrow.

**Comment by foxbit19** [comment]
Thanks in advance. I'll test the library when the fix will ready.

**Comment by oliverox** [comment]
@foxbit19 The issue has been resolved in version 1.0.1. You can just do an `npm update` and gatsby should build it fine this time. Let me know if you encounter any other issues.
